types/vote.go

```diff
@@ -53,16 +53,32 @@ type Address = crypto.Address
 // Vote represents a prevote, precommit, or commit vote from validators for
 // consensus.
 type Vote struct {
-	Type               cmtproto.SignedMsgType `json:"type"`
-	Height             int64                  `json:"height"`
-	Round              int32                  `json:"round"`    // assume there will not be greater than 2_147_483_647 rounds
-	BlockID            BlockID                `json:"block_id"` // zero if vote is nil.
-	Timestamp          time.Time              `json:"timestamp"`
-	ValidatorAddress   Address                `json:"validator_address"`
-	ValidatorIndex     int32                  `json:"validator_index"`
-	Signature          []byte                 `json:"signature"`
-	Extension          []byte                 `json:"extension"`
-	ExtensionSignature []byte                 `json:"extension_signature"`
+	Type             cmtproto.SignedMsgType `json:"type"`
```

Comment:
Please revert this hunk. No reason for keeping it different (if we hard-fork)

Reply:
Sure.  I guess there won't be any need of `LegacyVote` struct, which is taken from peppermint, right ? With hardfork it shouldn't be but confirming once.

Reply:
Correct. As we hard-fork, no need to backwards compatibility in the code base

Reply:
Resolved